src/hooks/useToggle.ts

```diff
@@ -2,7 +2,15 @@ import * as React from 'react';
 
 const useToggle = (initialState = false): [boolean, () => void] => {
   const [isToggled, setIsToggled] = React.useState(initialState);
-  const toggle = React.useCallback(() => setIsToggled((oldState) => !oldState), []);
+  const toggle = React.useCallback(
+    () =>
+      setIsToggled((oldState) => {
+        console.log('oldState', oldState);
```

Comment:
console log is forgotten here

Reply:
This has been fixed [here](https://github.com/Orfium/orfium-ictinus/pull/524/files)